test/gui/tst_checkAlltabs/test.feature

```diff
@@ -8,13 +8,13 @@ Feature: Visually check all tabs
     Scenario: Tabs in toolbar looks correct
         Given user "Alice" has been created on the server with default attributes and without skeleton files
         And user "Alice" has set up a client with default settings
-        Then the following tabs in the toolbar should match the default baseline
-            | AddAccount   |
-            | Activity     |
-            | Settings     |
-            | QuitOwncloud |
-
+        Then the toolbar should have the following tabs:
+            | Add Account |
+            | Activity    |
+            | Settings    |
+            | Quit        |
 
+    # Q: log dialog is not opened
     Scenario: Open log dialog with Ctrl+l keys combination
```

Comment:
`ctrl+l` doesn't open the log dialog. was it changed or regression?
CC @TheOneRing

Reply:
Removed on purpose.
The dialog is accessible through the settings tab.
I'll open an issue to update doc.